install.sh

```diff
@@ -0,0 +1,192 @@
+#!/bin/bash
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+get_script_dir () {
+     SOURCE="${BASH_SOURCE[0]}"
+     # While $SOURCE is a symlink, resolve it
+     while [ -h "$SOURCE" ]; do
+          DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+          SOURCE="$( readlink "$SOURCE" )"
+          [[ $SOURCE != /* ]] && SOURCE="$DIR/$SOURCE"
+     done
+     DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+     echo "$DIR"
+}
+
+root=$(get_script_dir)
+
+if ! [ -f "/usr/bin/clang++" ]; then
+    >&2 echo "Error: could not find clang++"
+    exit 8
+fi
+
+# Checking that we have everything
+system=$(uname -s)
+
+if [ "Linux" == $system ]; then
+    linuxGnuLibDir="/usr/lib/x86_64-linux-gnu"
```

Comment:
How universal is this path among Linux variants?  It doesn't look familiar to me (not currently sitting at a Linux box to check).